packages/react-components/src/unstable/package.json

```diff
@@ -4,5 +4,12 @@
   "module": "../lib/unstable/index.js",
   "typings": "../lib/unstable/index.d.ts",
   "sideEffects": false,
-  "license": "MIT"
+  "license": "MIT",
+  "exports": {
+    ".": {
+      "types": "./../lib/unstable/index.d.ts",
```

Comment:
Does `../` not work?

Reply:
auto generation code didn't minimize the path. The important part is that it starts with a period to indicate a relative path. I can fix

Reply:
I tried fixing this but I get a package.json validation error that says the paths much start with `./`.